<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LiuyaoResultFrame</class>
 <widget class="QFrame" name="LiuyaoResultFrame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>281</width>
    <height>516</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::StyledPanel</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Raised</enum>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>1</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>求测人</string>
     </property>
     <property name="buddy">
      <cstring>edit_name</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="edit_name">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>1</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QTextEdit" name="text_gua">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>4</verstretch>
      </sizepolicy>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>1</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>占事</string>
     </property>
     <property name="buddy">
      <cstring>edit_question</cstring>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QPlainTextEdit" name="text_analyze">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>1</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="edit_question">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>1</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string/>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="btn_add">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>新增记录</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_update">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>更新记录</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_del">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>删除记录</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btn_add</sender>
   <signal>clicked()</signal>
   <receiver>LiuyaoResultFrame</receiver>
   <slot>onBtnAdd()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>76</x>
     <y>480</y>
    </hint>
    <hint type="destinationlabel">
     <x>5</x>
     <y>382</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_update</sender>
   <signal>clicked()</signal>
   <receiver>LiuyaoResultFrame</receiver>
   <slot>onBtnUpdate()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>168</x>
     <y>482</y>
    </hint>
    <hint type="destinationlabel">
     <x>4</x>
     <y>342</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_del</sender>
   <signal>clicked()</signal>
   <receiver>LiuyaoResultFrame</receiver>
   <slot>onBtnDelete()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>249</x>
     <y>481</y>
    </hint>
    <hint type="destinationlabel">
     <x>4</x>
     <y>287</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>text_analyze</sender>
   <signal>textChanged()</signal>
   <receiver>LiuyaoResultFrame</receiver>
   <slot>onModified()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>222</x>
     <y>408</y>
    </hint>
    <hint type="destinationlabel">
     <x>275</x>
     <y>410</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>edit_name</sender>
   <signal>textChanged(QString)</signal>
   <receiver>LiuyaoResultFrame</receiver>
   <slot>onModified()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>186</x>
     <y>24</y>
    </hint>
    <hint type="destinationlabel">
     <x>276</x>
     <y>25</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>edit_question</sender>
   <signal>textChanged(QString)</signal>
   <receiver>LiuyaoResultFrame</receiver>
   <slot>onModified()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>213</x>
     <y>43</y>
    </hint>
    <hint type="destinationlabel">
     <x>276</x>
     <y>63</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onBtnAdd()</slot>
  <slot>onBtnUpdate()</slot>
  <slot>onBtnDelete()</slot>
  <slot>onModified()</slot>
 </slots>
</ui>
